File: DemoProject/DemoPreoject/CodeFeed/EntityCode/ModuleAcct/acct/V/BaseAcctViewIndex.m
//
//  BaseAcctViewIndex.m
//  ExampleProjectDemo
//
//  Created by BlanBok on 2018/1/17.
//  Copyright © 2018年 juyuanGroup. All rights reserved.
//

#import "BaseAcctViewIndex.h"

@implementation BaseAcctViewIndex

- (instancetype)init
{
    self = [super init];
    if (self) {
        [self subView_load];
    }
    return self;
}

- (void)setContainer:(UIView *)container
{
    _container = container;
    
    [self subView_layout];
}

#pragma mark-subView
- (void)subView_load
{
    
}

- (void)subView_layout
{
    
}
#pragma mark-loadSubView

@end

File: DemoPreoject/DemoPreoject/CodeFeed/EntityCode/ModuleAcct/Acct/V/index/BaseAcctViewIndex.swift
//
//  BaseAcctViewIndex.swift
//  MenuTreaure
//
//  Created by Mac on 2018/7/17.
//  Copyright © 2018年 OComme. All rights reserved.
//

import UIKit

class BaseAcctViewIndex: NSObject {
    /// 视图容器
    private var containerView:UIView!
    /// 设置容器
    func setContainerView(_ container:UIView) {
        self.containerView = container
        
        self.subView_load()
        self.subView_layout()
    }
    
    let view_head = BaseAcctInfoView();
    
    private func subView_load() {
        
        self.containerView.addSubview(self.view_head)

    }
    
    private func subView_layout() {
        
        view_head.snp.makeConstraints { (make) in
            make.top.left.right.equalTo(0)
        }
    }
}
